### DISCUSSION TITLE
Help dumping Nintendo Power Cartridge

### DISCUSSION THREAD

**Discussion by SggCnn93** [sanni/cartreader#802] [Need help]
Hi, I was trying to dump a Nintendo Power Cartridge with Picross NP Vol.1 and Vol.2 but I'm not having any success.
I tried googling and I found on reddit a link to the wiki about Nintendo Power Games but it seems the page doesn't exist anymore.
If I try dumping the game and test the rom it seems it only dumps the menu, if I try to start a game from the menu I get a black screen and nothing happens. The cartridge works as it should on a Super Famicom.
I tried dumping the cartridge on a Cart Reader V5.
Does anyone know the correct procedure to dump this cartridges or has a working link to the wiki page about Nintendo Power Games?
Thank you very much :)

**Comment by splash5** [comment]
You need to select "SF Memory Cassette" option inside "Super Nintendo/SFC", then OSCR will show "Game Menu" and "Flash Menu".
Select "Game Menu" then it will show all the game in NP cartridge, first will be "SHVC-MENU" which is the menu itself.
Select which game you want to read out then it will bring out a menu with few options which should self explain.

**Reply by SggCnn93** [reply]
Ty very much! Worked perfectly :D
